cime_config/config_grids.xml

```diff
@@ -2885,6 +2915,12 @@
       <desc>mpas.gis1to10km is a variable-resolution, from 1- to 10-km, MALI grid of the Greenland Ice Sheet.</desc>
     </domain>
 
+    <domain name="mpas.gis1to10kmR2">
```

Comment:
I don't know. We can certainly remove the support for this if that's what everyone agrees to. One reason I thought about leaving it in there for at least the near future is because I thought it could be a useful benchmark in terms of showing the performance and "scientific validity" (e.g., init. cond. optimization) improvements that have resulted from both this new init. cond. and the other code changes.

Reply:
Ok, that seems a valid reason to keep it for now then.